frontend/src/app/components/blockchain-blocks/blockchain-blocks.component.html

```diff
@@ -32,8 +32,8 @@
             </ng-template>
             <div [attr.data-cy]="'bitcoin-block-' + offset + '-index-' + i + '-fee-span'" class="fee-span"
               *ngIf="block?.extras?.feeRange; else emptyfeespan">
-              {{ block?.extras?.feeRange?.[0] | number:feeRounding }} - {{
-              block?.extras?.feeRange[block?.extras?.feeRange?.length - 1] | number:feeRounding }} <ng-container
+              {{ getMinBlockFee(block) | number:feeRounding }} - {{
```

Comment:
Is it possible to refactor this into not using function calls from the angular template. The problem is that the function is executed 500 times on load and 160 times every second to check for changes.

Reply:
yes, I've just moved that logic out of the template and into the data pipeline.